cef/CustomEditField/CustomEditField.rbbas

```diff
@@ -1462,7 +1470,8 @@ Implements MessageReceiver
 		      // but on Mac OS it's not needed any more.
 		      // In fact, it would prevent Retina / HiDPI rendering from working. Therefore, for
 		      // Mac builds, we now draw directly into the Canvas by not creating this back buffer
-		      mBackBuffer = new Picture(gr.Width, gr.Height, 32)
+		      //mBackBuffer = new Picture(gr.Width, gr.Height, 32)
+		      mBackBuffer = parentWindow.BitmapForCaching(gr.Width, gr.Height)
```

Comment:
This does not compile with RB 2012r2.1, which I want to keep working. Please add a #if RBVersion < 2013 where you use the old code and use your new only in the #else part.